polyply/src/generate_templates.py

```diff
@@ -308,9 +311,9 @@ def _gen_templates(self, meta_molecule):
                     if success:
                         break
                     elif opt_counter > self.max_opt:
-                        print("WARNING: Failed to optimize structure for block {}.".format(resname))
-                        print("         Proceeding with unoptimized coordinates.")
-                        print("         Usually this is OK, but check your final structure.")
+                        LOGGER.warning(("WARNING: Failed to optimize structure for block {}."
+                                        "Proceeding with unoptimized coordinates."
+                                        "Usually this is OK, but check your final structure.", resname))
```

Comment:
Remove the WARNING from the string. That should now be taken care of by the logging formatter